appraisal.lic

```diff
@@ -39,6 +41,15 @@ class Appraisal
     @equipment_manager.items.each { |item| break unless appraise_item?(item) }
   end
 
+  def train_appraisal_with_bundle
+    case bput("appraise my bundle quick", 'Roundtime', 'Appraise what', 'You cannot appraise')
```

Comment:
Whats the difference between using quick and not using it?

Reply:
it's harder so should train higher